evap/staff/templates/staff_user_merge.html

```diff
@@ -143,6 +143,42 @@ <h3>{% trans 'Merge users' %}</h3>
                             {% endfor %}
                         </td>
                     </tr>
+                    <tr{% if 'courses_responsible_for' in errors %} class="table-danger"{% endif %}>
+                        <td><strong>{% trans 'Courses responsible for' %}</strong></td>
+                        {% regroup main_user.get_sorted_courses_responsible_for by semester as course_list %}
+                        <td{% if course_list|length > 0 %} class="info"{% endif %}>
+                            {% for courses in course_list %}
+                                {{ courses.grouper }}
+                                <ul>
+                                    {% for course in courses.list %}
+                                        <li>{{ course.name }}</li>
+                                    {% endfor %}
+                                </ul>
+                            {% endfor %}
+                        </td>
+                        {% regroup other_user.get_sorted_courses_responsible_for by semester as course_list %}
+                        <td{% if course_list|length > 0 %} class="info"{% endif %}>
+                            {% for courses in course_list %}
+                                {{ courses.grouper }}
+                                <ul>
+                                    {% for course in courses.list %}
+                                        <li>{{ course.name }}</li>
+                                    {% endfor %}
+                                </ul>
+                            {% endfor %}
+                        </td>
+                        {% regroup merged_user.courses_responsible_for by semester as course_list %}
+                        <td{% if course_list|length > 0 %} class="success"{% endif %}>
+                            {% for courses in course_list %}
```

Comment:
we have this duplicated here two times - maybe include from another file?

Reply:
most things in this file exist three times. so that might be a refactoring idea for a different issue?

Reply:
Fair enough, created #1299